src/IPM/utils.jl

```diff
@@ -29,10 +24,26 @@ MadNLPExecutionStats(solver::MadNLPSolver) =MadNLPExecutionStats(
     solver.cnt.k, get_counters(solver.nlp),solver.cnt.total_time
 )
 
+function reset!(stats::MadNLPExecutionStats, solver::MadNLPSolver)
```

Comment:
I would suggest to rename `reset!` as `update!`: `reset!` would mean we reset `stats` to some default null values, whereas here we are copying the information from `solver` to `stats`.

Reply:
you're right! I'll update this